recipes/dowhy/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "dowhy" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2e59e4f2cb864c73cd87a84988bbda73fdc0aad19075306e383d520d4f8cbed4 
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - matplotlib
```

Please depend on `matplotlib-base` if you don't explicitly require the Qt backend.
```suggestion
    - matplotlib-base
```